### PR TITLE
[refine](profilev2) format profilev2

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

pipelineX

```
SimpleProfile:
    Instance(1):
        RESULT_SINK_OPERATOR  (id=0):
              -  CloseTime:  avg  11.729us,  max  11.729us,  min  11.729us
              -  InputRows:  0
              -  OpenTime:  avg  99.257us,  max  99.257us,  min  99.257us
        AGGREGATION_OPERATOR  (id=793):
              -  PlanInfo
                    -  output:  sum(partial_sum((lo_extendedprice  *  lo_discount)))[#45]
                    -  group  by:  
                    -  cardinality=1
              -  BlocksReturned:  1
              -  CloseTime:  avg  6.361us,  max  6.361us,  min  6.361us
              -  OpenTime:  avg  24.928us,  max  24.928us,  min  24.928us
              -  ProjectionTime:  avg  0ns,  max  0ns,  min  0ns
              -  RowsReturned:  1
        AGGREGATION_SINK_OPERATOR  (id=793):
              -  CloseTime:  avg  2.975us,  max  2.975us,  min  2.975us
              -  InputRows:  1
              -  OpenTime:  avg  86.625us,  max  86.625us,  min  86.625us
        EXCHANGE_OPERATOR  (id=790):
              -  PlanInfo
                    -  offset:  0
              -  BlocksReturned:  1
              -  CloseTime:  avg  13.550us,  max  13.550us,  min  13.550us
              -  OpenTime:  avg  88.58us,  max  88.58us,  min  88.58us
              -  ProjectionTime:  avg  0ns,  max  0ns,  min  0ns
              -  RowsReturned:  1
    Instance(1):
        DATA_STREAM_SINK_OPERATOR  (id=790,dest_id=790):
              -  BlocksSent:  1
              -  CloseTime:  avg  45.10us,  max  45.10us,  min  45.10us
              -  InputRows:  1
              -  OpenTime:  avg  164.442us,  max  164.442us,  min  164.442us
        AGGREGATION_OPERATOR  (id=787):
              -  PlanInfo
                    -  output:  partial_sum((lo_extendedprice  *  lo_discount))[#44]
                    -  group  by:  
                    -  cardinality=1
              -  BlocksReturned:  1
              -  CloseTime:  avg  6.610us,  max  6.610us,  min  6.610us
              -  OpenTime:  avg  21.107us,  max  21.107us,  min  21.107us
              -  ProjectionTime:  avg  0ns,  max  0ns,  min  0ns
              -  RowsReturned:  1
        AGGREGATION_SINK_OPERATOR  (id=787):
              -  CloseTime:  avg  3.892us,  max  3.892us,  min  3.892us
              -  InputRows:  11.913972M  (11913972)
              -  OpenTime:  avg  86.507us,  max  86.507us,  min  86.507us
        HASH_JOIN_OPERATOR  (id=781):
              -  PlanInfo
                    -  join  op:  INNER  JOIN(BUCKET_SHUFFLE)[]
                    -  equal  join  conjunct:  d_datekey  =  lo_orderdate
                    -  runtime  filters:  RF000[in_or_bloom]  <-  lo_orderdate(-1/0/2097152)
                    -  cardinality=1
                    -  vec  output  tuple  id:  5
                    -  vIntermediate  tuple  ids:  4  
                    -  hash  output  slot  ids:  18  19  
              -  BlocksReturned:  2.909K  (2909)
              -  CloseTime:  avg  11.679us,  max  11.679us,  min  11.679us
              -  OpenTime:  avg  87.69us,  max  87.69us,  min  87.69us
              -  ProbeRows:  365
              -  ProjectionTime:  avg  22.279ms,  max  22.279ms,  min  22.279ms
              -  RowsReturned:  11.913972M  (11913972)
            OLAP_SCAN_OPERATOR  (id=760):
                  -  PlanInfo
                        -  TABLE:  default_cluster:ssb.dates(dates),  PREAGGREGATION:  ON
                        -  PREDICATES:  d_year  =  1993
                        -  runtime  filters:  RF000[in_or_bloom]  ->  d_datekey
                        -  partitions=1/1,  tablets=1/1,  tabletList=11778
                        -  cardinality=1,  avgRowSize=0.0,  numNodes=1
                        -  pushAggOp=NONE
                        -  projections:  d_datekey
                        -  project  output  tuple  id:  3
                  -  BlocksReturned:  1
                  -  CloseTime:  avg  123.533us,  max  123.533us,  min  123.533us
                  -  OpenTime:  avg  731.258us,  max  731.258us,  min  731.258us
                  -  ProjectionTime:  avg  71.935us,  max  71.935us,  min  71.935us
                  -  RowsReturned:  365
                :
                VScanner:
                    SegmentIterator:
        HASH_JOIN_SINK_OPERATOR  (id=781):
              -  CloseTime:  avg  0ns,  max  0ns,  min  0ns
              -  InputRows:  78.550619M  (78550619)
              -  OpenTime:  avg  155.948us,  max  155.948us,  min  155.948us
        EXCHANGE_OPERATOR  (id=778):
              -  PlanInfo
                    -  offset:  0
              -  BlocksReturned:  23.621K  (23621)
              -  CloseTime:  avg  18.633us,  max  18.633us,  min  18.633us
              -  OpenTime:  avg  67.734us,  max  67.734us,  min  67.734us
              -  ProjectionTime:  avg  0ns,  max  0ns,  min  0ns
              -  RowsReturned:  78.550619M  (78550619)
    Instance(8):
        DATA_STREAM_SINK_OPERATOR  (id=778,dest_id=778):
              -  BlocksSent:  23.621K  (23621)
              -  CloseTime:  avg  43.670us,  max  71.842us,  min  34.916us
              -  InputRows:  78.550619M  (78550619)
              -  OpenTime:  avg  110.840us,  max  124.442us,  min  103.910us
        OLAP_SCAN_OPERATOR  (id=767):
              -  PlanInfo
                    -  TABLE:  default_cluster:ssb.lineorder(lineorder),  PREAGGREGATION:  ON
                    -  PREDICATES:  lo_discount  <=  3  AND  lo_discount  >=  1  AND  lo_quantity  <  25
                    -  partitions=7/7,  tablets=336/336,  tabletList=11074,11076,11078  ...
                    -  cardinality=1,  avgRowSize=0.0,  numNodes=1
                    -  pushAggOp=NONE
                    -  projections:  lo_orderdate,  lo_extendedprice,  lo_discount
                    -  project  output  tuple  id:  1
              -  BlocksReturned:  23.621K  (23621)
              -  CloseTime:  avg  299.38us,  max  436.140us,  min  240.322us
              -  OpenTime:  avg  2.514ms,  max  3.680ms,  min  1.703ms
              -  ProjectionTime:  avg  80.705ms,  max  85.633ms,  min  71.923ms
              -  RowsReturned:  78.550619M  (78550619)
            VScanner:
                SegmentIterator:
```

pipeline
```
SimpleProfile:
    Instance(1):
        VDataBufferSender  (dst_fragment_instance_id=-6e2d87aa85b6b5ff--52848c53948863f5):
        VAGGREGATION_NODE  (id=789):
              -  PlanInfo
                    -  output:  sum(partial_sum((lo_extendedprice  *  lo_discount)))[#45]
                    -  group  by:  
                    -  cardinality=1
              -  BlockCount:  1
              -  BuildTime:  avg  0ns,  max  0ns,  min  0ns
              -  ExecTime:  avg  254.705us,  max  254.705us,  min  254.705us
              -  OutputBytes:  4.03  KB
              -  RowsReturned:  1
            VEXCHANGE_NODE  (id=786):
                  -  PlanInfo
                        -  offset:  0
                  -  BlockCount:  8
                  -  ExecTime:  avg  153.349us,  max  153.349us,  min  153.349us
                  -  OutputBytes:  512.00  B
                  -  RowsReturned:  8
    Instance(8):
        VDataStreamSender  (dst_id=786,  dst_fragments=[{"91d278557a494a01-ad7b73ac6b779c0b"}]):
              -  BlocksSent:  8
        VAGGREGATION_NODE  (id=783):
              -  PlanInfo
                    -  output:  partial_sum((lo_extendedprice  *  lo_discount))[#44]
                    -  group  by:  
                    -  cardinality=1
              -  BlockCount:  8
              -  BuildTime:  avg  82.67ms,  max  117.939ms,  min  19.573ms
              -  ExecTime:  avg  123.348ms,  max  176.956ms,  min  29.843ms
              -  OutputBytes:  512.00  B
              -  RowsReturned:  8
            VHASH_JOIN_NODE  (id=777):
                  -  PlanInfo
                        -  join  op:  INNER  JOIN(BROADCAST)[]
                        -  equal  join  conjunct:  lo_orderdate  =  d_datekey
                        -  runtime  filters:  RF000[in_or_bloom]  <-  d_datekey(-1/0/2097152)
                        -  cardinality=75,006,015
                        -  vec  output  tuple  id:  5
                        -  vIntermediate  tuple  ids:  4  
                        -  hash  output  slot  ids:  36  37  
                  -  BlockCount:  3.575K  (3575)
                  -  ExecTime:  avg  683.521ms,  max  971.39ms,  min  144.256ms
                  -  OutputBytes:  120.94  MB
                  -  RowsReturned:  11.913972M  (11913972)
                ProbePhase:
                      -  ProbeRows:  11.913972M  (11913972)
                      -  ProbeTime:  avg  682.178ms,  max  969.182ms,  min  143.805ms
                BuildPhase:
                      -  BuildRows:  365
                      -  BuildTime:  avg  67.643us,  max  541.151us,  min  0ns
                VEXCHANGE_NODE  (id=774):
                      -  PlanInfo
                            -  offset:  0
                      -  BlockCount:  1
                      -  ExecTime:  avg  65.314us,  max  77.500us,  min  48.296us
                      -  OutputBytes:  2.00  KB
                      -  RowsReturned:  365
                VNewOlapScanNode(lineorder)  (id=756):
                      -  PlanInfo
                            -  TABLE:  default_cluster:ssb.lineorder(lineorder),  PREAGGREGATION:  ON
                            -  PREDICATES:  lo_discount  <=  3  AND  lo_discount  >=  1  AND  lo_quantity  <  25
                            -  runtime  filters:  RF000[in_or_bloom]  ->  lo_orderdate
                            -  partitions=7/7,  tablets=336/336,  tabletList=11074,11076,11078  ...
                            -  cardinality=600037902,  avgRowSize=130.72264,  numNodes=1
                            -  pushAggOp=NONE
                            -  projections:  lo_orderdate,  lo_extendedprice,  lo_discount
                            -  project  output  tuple  id:  3
                      -  BlockCount:  3.575K  (3575)
                      -  ExecTime:  avg  15.487ms,  max  21.927ms,  min  4.504ms
                      -  OutputBytes:  174.52  MB
                      -  RowsReturned:  11.913972M  (11913972)
                    :
                    VScanner:
                        SegmentIterator:
    Instance(8):
        VDataStreamSender  (dst_id=774,  dst_fragments=[{"91d278557a494a01-ad7b73ac6b779c03"}]):
              -  BlocksSent:  1
        VNewOlapScanNode(dates)  (id=765):
              -  PlanInfo
                    -  TABLE:  default_cluster:ssb.dates(dates),  PREAGGREGATION:  ON
                    -  PREDICATES:  d_year  =  1993
                    -  partitions=1/1,  tablets=1/1,  tabletList=11778
                    -  cardinality=2556,  avgRowSize=68.56221,  numNodes=1
                    -  pushAggOp=NONE
                    -  projections:  d_datekey
                    -  project  output  tuple  id:  1
              -  BlockCount:  1
              -  ExecTime:  avg  258.666us,  max  411.533us,  min  180.518us
              -  OutputBytes:  2.00  KB
              -  RowsReturned:  365
            VScanner:
                SegmentIterator:
```

no pipeline
```
SimpleProfile:
    Instance(1):
        VDataBufferSender  (dst_fragment_instance_id=104524f196a845a0--7ca35c7d54b3254e):
        VAGGREGATION_NODE  (id=789):
              -  PlanInfo
                    -  output:  sum(partial_sum((lo_extendedprice  *  lo_discount)))[#45]
                    -  group  by:  
                    -  cardinality=1
              -  BlockCount:  1
              -  BuildTime:  avg  0ns,  max  0ns,  min  0ns
              -  ExecTime:  avg  245.579us,  max  245.579us,  min  245.579us
              -  OutputBytes:  4.03  KB
              -  RowsReturned:  1
            VEXCHANGE_NODE  (id=786):
                  -  PlanInfo
                        -  offset:  0
                  -  BlockCount:  8
                  -  ExecTime:  avg  1s132ms,  max  1s132ms,  min  1s132ms
                  -  OutputBytes:  512.00  B
                  -  RowsReturned:  8
    Instance(8):
        VDataStreamSender  (dst_id=786,  dst_fragments=[{"104524f196a845a0-835ca382ab4cdab2"}]):
              -  BlocksSent:  8
        VAGGREGATION_NODE  (id=783):
              -  PlanInfo
                    -  output:  partial_sum((lo_extendedprice  *  lo_discount))[#44]
                    -  group  by:  
                    -  cardinality=1
              -  BlockCount:  8
              -  BuildTime:  avg  81.278ms,  max  109.181ms,  min  25.462ms
              -  ExecTime:  avg  122.42ms,  max  163.947ms,  min  38.339ms
              -  OutputBytes:  512.00  B
              -  RowsReturned:  8
            VHASH_JOIN_NODE  (id=777):
                  -  PlanInfo
                        -  join  op:  INNER  JOIN(BROADCAST)[]
                        -  equal  join  conjunct:  lo_orderdate  =  d_datekey
                        -  runtime  filters:  RF000[in_or_bloom]  <-  d_datekey(-1/0/2097152)
                        -  cardinality=75,006,015
                        -  vec  output  tuple  id:  5
                        -  vIntermediate  tuple  ids:  4  
                        -  hash  output  slot  ids:  36  37  
                  -  BlockCount:  3.575K  (3575)
                  -  ExecTime:  avg  770.537ms,  max  986.275ms,  min  265.347ms
                  -  OutputBytes:  120.09  MB
                  -  RowsReturned:  11.913972M  (11913972)
                ProbePhase:
                      -  ProbeRows:  11.913972M  (11913972)
                      -  ProbeTime:  avg  767.581ms,  max  982.862ms,  min  262.871ms
                BuildPhase:
                      -  BuildRows:  365
                      -  BuildTime:  avg  62.703us,  max  501.628us,  min  0ns
                VEXCHANGE_NODE  (id=774):
                      -  PlanInfo
                            -  offset:  0
                      -  BlockCount:  1
                      -  ExecTime:  avg  245.797us,  max  1.610ms,  min  48.227us
                      -  OutputBytes:  2.00  KB
                      -  RowsReturned:  365
                VNewOlapScanNode(lineorder)  (id=756):
                      -  PlanInfo
                            -  TABLE:  default_cluster:ssb.lineorder(lineorder),  PREAGGREGATION:  ON
                            -  PREDICATES:  lo_discount  <=  3  AND  lo_discount  >=  1  AND  lo_quantity  <  25
                            -  runtime  filters:  RF000[in_or_bloom]  ->  lo_orderdate
                            -  partitions=7/7,  tablets=336/336,  tabletList=11074,11076,11078  ...
                            -  cardinality=600037902,  avgRowSize=130.72264,  numNodes=1
                            -  pushAggOp=NONE
                            -  projections:  lo_orderdate,  lo_extendedprice,  lo_discount
                            -  project  output  tuple  id:  3
                      -  BlockCount:  3.575K  (3575)
                      -  ExecTime:  avg  33.403ms,  max  36.769ms,  min  24.684ms
                      -  OutputBytes:  180.14  MB
                      -  RowsReturned:  11.913972M  (11913972)
                    :
                    VScanner:
                        SegmentIterator:
    Instance(1):
        VDataStreamSender  (dst_id=774,  dst_fragments=[{"104524f196a845a0-835ca382ab4cdaaa",  "104524f196a845a0-835ca382ab4cdaab",  "104524f196a845a0-835ca382ab4cdaac",  "104524f196a845a0-835ca382ab4cdaad",  "104524f196a845a0-835ca382ab4cdaae",  "104524f196a845a0-835ca382ab4cdaaf",  "104524f196a845a0-835ca382ab4cdab0",  "104524f196a845a0-835ca382ab4cdab1"}]):
              -  BlocksSent:  1
        VNewOlapScanNode(dates)  (id=765):
              -  PlanInfo
                    -  TABLE:  default_cluster:ssb.dates(dates),  PREAGGREGATION:  ON
                    -  PREDICATES:  d_year  =  1993
                    -  partitions=1/1,  tablets=1/1,  tabletList=11778
                    -  cardinality=2556,  avgRowSize=68.56221,  numNodes=1
                    -  pushAggOp=NONE
                    -  projections:  d_datekey
                    -  project  output  tuple  id:  1
              -  BlockCount:  1
              -  ExecTime:  avg  1.965ms,  max  1.965ms,  min  1.965ms
              -  OutputBytes:  2.00  KB
              -  RowsReturned:  365
            VScanner:
                SegmentIterator:
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/common/util/RuntimeProfile.java [hunk 1]
```diff
@@ -690,7 +820,9 @@ public void setProfileLevel(int profileLevel) {
     }
 
     public void setPlaner(Planner planner) {
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/RuntimeProfile.java:822]: not use set planner, rename to setFragmentPlanInfo(map<id,info>)

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Review by wm1581066 — APPROVED** [review]
LGTM

**Pull request merged**